Worksheet.md

```diff
@@ -27,6 +26,10 @@
 
 If you need any help with this, then feel free to ask one of our mentors for help!
 
+### What is Linux?
+
+The term 'Linux' refers to the 'kernel' that the system is running. The kernel has control over the whole system, and connects the application software on a computer to the CPU, memory and input/output devices. There are [literally hundreds](https://en.wikipedia.org/wiki/List_of_Linux_distributions) of linux distributions, and many more instances of linux running on embedded devices like in-flight entertainment systems, digital billboards, and other such things. The ethos behind the Linux ecosystem is having a 'free and open source' codebase, which means that the source code is public, and anybody is free to change it and re-release it (Under a different name). 
```

Comment:
Try to keep your capitalisation of Linux consistent, also try simplifying your explanation of what a kernel is slightly in the interest of keeping this worksheet accessible to non-CS people :)